examples/constrained/multiclass_linear_svm.py

```diff
@@ -102,8 +102,8 @@ def matvec_G(_, beta):
                       params_eq=(None, b),
                       params_ineq=(None, h))
   
-  init_params = osqp._box_osqp.init_params(init_x=None, **hyper_params)
-  sol, _ = osqp.run(init_params=None, **hyper_params)
+  init_params = osqp.init_params(init_x=None, **hyper_params)
```

Comment:
Unrelated to this example, but why is it called `init_x`? Shouldn't this be called `init_params` instead?

Reply:
That would make sense indeed.